vnext/ReactUWP/Modules/NativeUIManager.cpp

```diff
@@ -187,6 +188,11 @@ void NativeUIManager::AddRootView(
   XamlView view = xamlRootView->GetXamlView();
   m_tagsToXamlReactControl.emplace(shadowNode.m_tag, xamlRootView->GetXamlReactControl());
 
+  // Push the appropriate FlowDirection into the root view.
+  auto flow = I18nHelper::Instance().getIsRTL() ? winrt::FlowDirection::RightToLeft : winrt::FlowDirection::LeftToRight;
+  view.as<winrt::FrameworkElement>().FlowDirection(
+      I18nHelper::Instance().getIsRTL() ? winrt::FlowDirection::RightToLeft : winrt::FlowDirection::LeftToRight);
```

Comment:
can you please use the updated namespaces? (xaml::FrameworkElement, etc.)

Reply:
done

---
In reply to: [416489965](https://github.com/microsoft/react-native-windows/pull/4732#discussion_r416489965) [](ancestors = 416489965)